config/settings/base.py

```diff
@@ -237,3 +240,24 @@
 CLIENT_VERIFY_CERTIFICATES = strtobool(
     os.getenv("ROSETTA_CLIENT_VERIFY_CERTIFICATES", "True")
 )
+
+
+SENTRY_DEBUG_URL_ENABLED = False
+if SENTRY_DSN := os.getenv("SENTRY_DSN", ""):
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        environment=os.getenv("SENTRY_ENVIRONMENT", ""),
+        release=get_git_sha(),
+        integrations=[DjangoIntegration()],
+        # Set traces_sample_rate to 1.0 to capture 100%
+        # of transactions for performance monitoring.
+        # We recommend adjusting this value in production.
+        traces_sample_rate=float(os.getenv("SENTRY_SAMPLE_RATE", "0.5")),
```

Comment:
Set different default sample rates for all environments: https://github.com/search?q=repo%3Anationalarchives%2Fds-frontend%20SENTRY_SAMPLE_RATE&type=code

Reply:
Have also added `sample_rate`, `profiles_sample_rate` as done in `ds-frontend`